src/common/post.ts

```diff
@@ -202,3 +204,24 @@ export const validatePost = (args: ValidatePostArgs): ValidatePostArgs => {
 
   return { title, content };
 };
+
+export const getPostsTinybirdExport = (con: DataSource, latest: Date) =>
```

Comment:
Why did you decide to move this.
Maybe keeping it isolated in tinybird cron is more explicit as to what needs review from Viktor moving forward?

Just to keep the constraint of changes in that file need his approval.

Reply:
Agree